django/core/validators.py

```diff
@@ -104,7 +107,101 @@ class URLValidator(RegexValidator):
         r'(?:[/?#][^\s]*)?'  # resource path
         r'\Z', re.IGNORECASE)
     message = _('Enter a valid URL.')
-    schemes = ['http', 'https', 'ftp', 'ftps']
+    schemes = [
```

Comment:
Changing this list is outside the scope of this ticket and also backwards-incompatible.